components/src/molecules/dropdown-list-item/DropdownListItem.html

```diff
@@ -5,18 +5,22 @@
     @click="toggle"
```

Comment:
In this file
```
<li :class="['dropdown-list__item', {'dropdown-list__item--collapse': collapseLocal}]">
```
we should change it to 
```
<li class="dropdown-list__item">
```
and add collapse class to info

Reply:
But this variable is dependent on logic inside the component.